nservicebus/azure-storage-persistence/configuration_sections_azure_[,6.2].partial.md

```diff
@@ -0,0 +1,5 @@
+### Configuration with Configuration Section
```

Comment:
Edit: read your first comment here.

Why don't you use component name in all partials? I'll see now locally how it renders, interesting case :)